src/vea/vea_alloc.c

```diff
@@ -592,53 +599,74 @@ new_chunk_commit_cb(void *data, bool noop)
 	bitmap_entry->vbe_published_state = VEA_BITMAP_STATE_PUBLISHED;
 }
 
+static void
+new_chunk_abort_cb(void *data, bool noop)
```

Comment:
When 'noop' is true, the state changing should be avoided. (see umem_stage_callback()).

Reply:
Fixed, thanks!